task.go

```diff
@@ -22,7 +22,8 @@ const (
 // Task is a task. 🎊
 type Task struct {
 	ID              ID     `json:"id,omitempty"`
-	Organization    ID     `json:"orgID"`
+	OrganizationID  ID     `json:"orgID"`
+	Organization    string `json:"organization"`
```

Comment:
As above, I'm leaning towards `org` rather than `organization` for the JSON key.